login/service_blackbox_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 	"github.com/almighty/almighty-core/account"
 	"github.com/almighty/almighty-core/app"
-	"github.com/almighty/almighty-core/configuration"
+	config "github.com/almighty/almighty-core/configuration"
```

Comment:
for a matter of consistency, could you use the same `c` or `config` (I have no preference) as a package alias in all tests ?